forest/src/cli/snapshot_fetch.rs

```diff
@@ -49,15 +39,20 @@ pub(crate) async fn snapshot_fetch(
 /// Fetches snapshot for `calibnet` from a default, trusted location. On success, the snapshot will be
 /// saved in the given directory. In case of failure (e.g. connection interrupted) it will not be
 /// removed.
-async fn snapshot_fetch_calibnet(snapshot_out_dir: &Path) -> anyhow::Result<PathBuf> {
-    let bucket = Bucket::new_public(
-        DEFAULT_CALIBNET_BUCKET_NAME,
-        DEFAULT_CALIBNET_REGION.parse()?,
-    )?;
+async fn snapshot_fetch_calibnet(
+    snapshot_out_dir: &Path,
+    snapshot_fetch_config: &SnapshotFetchConfig,
+) -> anyhow::Result<PathBuf> {
+    let name = snapshot_fetch_config.calibnet.bucket_name.clone();
+    let region = snapshot_fetch_config.calibnet.region.clone();
+    let bucket = Bucket::new_public(name.as_str(), region.as_str().parse()?)?;
```

Comment:
Please consider using references instead of clones whenever it's possible. I know it's not a bottleneck here, but it's good practice not to have needless operations that don't necessarily improve readability.
```suggestion
    let name = &snapshot_fetch_config.calibnet.bucket_name;
    let region = &snapshot_fetch_config.calibnet.region;
    let bucket = Bucket::new_public(name, region.parse()?)?;
```

Reply:
Fixed